packages/design-system-tokens/src/themes/core-components.ts

```diff
@@ -248,7 +248,7 @@ export const components: AnyTokenValues = {
     '__border-color':                             t.color['base'],
     '__border-color--checked':                    t.color['primary'],
     '__border-color--disabled':                   t.color['gray-light'],
-    '__border-color--error':                      t.color['error'],
+    '__border-color--error':                      t.color['base'],
```

Comment:
Would some kind of comment like this be helpful? Or is a theme just full of decisions, and a comment would be noise?

```suggestion
    '__border-color--error':                      t.color['base'], // Don't use error border for choice in this theme
```

Reply:
Sure this works.